cddl/protected-corim-header-map.cddl

```diff
@@ -1,6 +1,16 @@
-protected-corim-header-map = {
-  &(alg: 1) => int
-  &(content-type: 3) => "application/rim+cbor"
-  &(corim-meta: 8) => bstr .cbor corim-meta-map
-  * cose-label => cose-value
-}
+; protected header map needs to contain at least one of corim-meta (8) or CWT-Claims (15)
+protected-corim-header-map =
+  {
+    &(alg: 1) => int,
+    &(content-type: 3) => "application/rim+cbor",
+    meta,
+    * cose-label => cose-value
+  }
+
+meta = ((
+        corim-meta-fragment,
```

Comment:
Why is corim-meta-fragment required when the text says either or both could be supplied?

Reply:
I think you are reading the CDDL wrong. Line 11-13 specifiy: either corim-meta-fragment, or cwt-claims-fragment, or both. corim-meta-fragment is not mandatory.